### PR TITLE
[CHANGELOG] Document recent work

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +42,12 @@ of tasks being currently executed.
 - JAMES-3122 Log4J2 adoption for Spring product. Log file configuration needs to be updated. See upgrade instructions.
 - JAMES-2760 mailqueue.size.metricsEnabled should be false by default
 - JAMES-3252 DomainList autoDetection should be turned off by default. Operators relying on implicit values for enabling DomainList autoDetection now needs to explicitly configure it.
+- JAMES-3295 Multiple IMAP performance enhancements for the Distributed Server. Some enhancement might transfer to other servers as well.
+- JAMES-3184 Throttling mechanism allow an admin to specify the throughput desired for a given WebAdmin task
```
Comment (rouazana) [inline CHANGELOG.md:46]:
```suggestion
- JAMES-3184 Throttling mechanism allows an admin to specify the throughput desired for a given WebAdmin task
```

**Comment by chibenwa** [comment]
`Caused by: java.io.FileNotFoundException: dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties (No such file or directory)`

Is this solved?

Cc @blackheaven ?

test this please

**Comment by chibenwa** [comment]
https://github.com/linagora/james-project/pull/3599 should allow us to debug deployment tests issues here.

**Comment by blackheaven** [comment]
it should not happen

**Comment by chibenwa** [comment]
```
RabbitMQDeletedMessageVaultIntegrationTest>DeletedMessageVaultIntegrationTest.vaultExportShouldExportZipContainsVaultMessagesToShareeWhenImapDeletedMailbox:543 
[65d194823aec50a75f95b00199334f305549ca73] Expecting zipFile to contains 1 entries but actually contains (0) entries
```
test this please

**Comment by chibenwa** [comment]
`Caused by: java.io.FileNotFoundException: null/dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties (No such file or directory)`

Marf

test this please

**Comment by Arsnael** [comment]
Do you really need to run the build on changelog changes?

**Comment by chibenwa** [comment]
> Do you really need to run the build on changelog changes?

No, not really, but it helps diagnosing the CI don't do its job.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**